src/main/java/frc/robot/subsystems/drive/Swerve.java

```diff
@@ -41,6 +54,14 @@ public class Swerve extends SubsystemBase {
     protected static final String LogKey = "Swerve";
     protected static final String OdometryLogKey = "Odometry";
 
+    private static final HolonomicPathFollowerConfig HolonomicPathFollowerConfig = new HolonomicPathFollowerConfig(
```

Comment:
If we aren't using pathplanner. We can remove this.

Reply:
Same reasoning as above. Also, the original idea was that if we wanted, we can always still use pathplanner if we needed to without doing any of the setup.